tests/spoke/slicegw_controller_test.go

```diff
@@ -338,7 +338,8 @@ var _ = Describe("Worker SlicegwController", func() {
 			}, time.Second*10, time.Millisecond*250).Should(BeTrue())
 
 			createdSliceGw.Status.Config.SliceGatewayHostType = "Client"
-
+			createdSliceGw.Status.Config.SliceGatewayRemoteNodeIP = "1.2.3.4"
+			createdSliceGw.Status.Config.SliceGatewayRemoteGatewayID = "remote-gateway-id"
 			Eventually(func() bool {
```

Comment:
Can you add test cases for the feature? Or do you want to do in a separate PR?

Reply:
yeah I will add this as a part of this PR